### DISCUSSION TITLE
All jpg's download as zero byte files.

### DISCUSSION THREAD

**Discussion by Someonemustnothavethis** [TumblThreeApp/TumblThree#368] [Q&A (Question & Answer)]
The last time I caught up with all the blogs (about 200 of them) it worked fine although a few images showed up as zero byte files.
This was the middle of last month.
I just tried to catch up again and ALL images download as zero byte files.  I didn't change anything.

**Comment by johanneszab** [comment]
Any chance you are on windows 7 and are not using the latest TumblThree version?

If so, look here (#74) and try the latest version. Otherwise, I can't give any help really with that information. Seems to working fine for me for regular and hidden blogs (Tumblr v1 API and Tumblr SVC) on Windows 10.

**Comment by Someonemustnothavethis** [comment]
Aha...it works now.   I'm not a programmer and so missed this.
 Thanks.
